### ISSUE TITLE
Is PlayHide secure?

### ISSUE THREAD

**Issue by Darthagnon** [DoM1niC/PlayHide#13]
When running PlayHide VPN, it looked like I could connect to other PCs as a LAN, without any password or check. What's to stop someone with malicious intent connecting to me and hacking me over the virtual LAN, since afaik they won't be prompted for a password to join the LAN?

Don't know if I'm misunderstanding how PlayHide VPN works completely?

**Comment by DoM1niC** [comment]
yep you are right, playhide is fully open wihout any login credentials and I improve soon the port forwarding for security based ports like Windows Samba / Remote Services, Passwort auth is possible but currently not want from my side.... The Connection is secured over a Cert this is necessary 4 VPN Connections.

**Comment by DoM1niC** [comment]
> Thank you for the response!
> 
> Isn't it risky to use? e.g. I connect to PlayHide VPN. Hacker connects to PlayHide VPN. He can see my PC as if on LAN (?), and can access my folders that I share to home LAN (?), and could install one of those ransomware viruses that spreads via LAN. And I can't stop him from joining the same PlayHide server that I'm connected to.
> 
> Is that a possibility at all?

My opinion is to check your Windows Settings, normally you musst setup network sharing manual... or you use your Windows Firewall to allow all ports if you want.... and restrict all services you do not use.

**Comment by Darthagnon** [comment]
Thank you so much for implementing the disabling SMB feature! I normally keep a few folders shared to my Home Network LAN (but not on Public networks). I need to double check how PlayHide work on my PC, to see whether it detects the connection as a Private or Public network.

**Comment by o0Julia0o** [comment]
Is ther an eable/disable SMB.bat? This would be useful.
